aws-predictions/src/main/java/com/amplifyframework/predictions/aws/service/AWSRekognitionService.java

```diff
@@ -133,9 +133,19 @@ void recognizeCelebrities(
             @NonNull Consumer<IdentifyResult> onSuccess,
             @NonNull Consumer<PredictionsException> onError
     ) {
+        final IdentifyEntitiesConfiguration config;
         try {
-            List<CelebrityDetails> celebrities = detectCelebrities(imageData);
-            onSuccess.accept(IdentifyCelebritiesResult.fromCelebrities(celebrities));
+            config = pluginConfiguration.getIdentifyEntitiesConfiguration();
+            if (config.isCelebrityDetectionEnabled()) {
+                List<CelebrityDetails> celebrities = detectCelebrities(imageData);
+                onSuccess.accept(IdentifyCelebritiesResult.fromCelebrities(celebrities));
+            } else {
+                onError.accept(new PredictionsException("Celebrity detection is disabled.",
```

Comment:
How about, `"Celebrity detection has not been configured."` Disabled sounds more like it has been turned off explicitly, or is has been made unavailable by policy.

Reply:
There is already an explicit "celebrityDetectionEnabled" flag under "identifyEntities" configuration, so I think this message is appropriate.